Common/VR/PPSSPPVR.cpp

```diff
@@ -201,6 +202,58 @@ void UpdateVRInput(bool(*NativeKey)(const KeyInput &key), bool(*NativeTouch)(con
 		}
 	}
 
+	//motion control
+	if (g_Config.bEnableMotions) {
```

Comment:
Come to think of it, do we really need an enable option? Wouldn't this just not do anything if no motion codes are mapped? Sure there's a little bit of computation here that can be saved by really not a lot.

Reply:
If people use a 3rd party controller then the movement could do some madness. This is there to avoid it.

Reply:
Okay, I suppose. Just trying to minimize the amount of options, as always...